Assets/Scripts/Minigames/CowardMiniGame/GearCompletionCheck.cs

```diff
@@ -0,0 +1,58 @@
+/*****************************************************************************
+// File Name :         GearCompletionCheck.cs
+// Author :            Mark Hanson
+// Creation Date :     5/27/2024
+//
+// Brief Description : Any function to do for the gears mini game will be found here. Includes swapping slots, Correct slot pattern with all bad ones, and selecting gears for each slot.
```

Comment:
Both this script and GearBehavior share this same description. Do they both serve similar functions?

Reply:
that was a blind site by me thank you for bringing that to my attention